purchases/src/main/kotlin/com/revenuecat/purchases/HTTPClient.kt

```diff
@@ -132,6 +133,8 @@ internal class HTTPClient(
                 Integer.toString(Build.VERSION.SDK_INT)
             )
             addRequestProperty("X-Version", Purchases.frameworkVersion)
+            addRequestProperty("X-Client-Locale", appConfig.languageTag)
```

Comment:
should we also add it to iOS for consistency? or just rely on `Accept-Language`?

Reply:
Or I can pass it as Accept-Language on Android instead?

Reply:
Mmmh no, I think it's fine. We would prioritize this one and then fallback to `Accept-Language` server side.

Reply:
yeah, having our own special header is more reliable